server/monitor.go

```diff
@@ -1813,12 +1817,7 @@ func (s *Server) HandleLeafz(w http.ResponseWriter, r *http.Request) {
 	if err != nil {
 		return
 	}
-	var opts *LeafzOptions
-	if subs {
-		opts = &LeafzOptions{Subscriptions: true}
-	}
-
-	l, err := s.Leafz(opts)
+	l, err := s.Leafz(&LeafzOptions{subs, r.URL.Query().Get("acc")})
```

Comment:
Wonder if we should have a `decodeString()` since I see that you start to use that in few places.
We could return an error if not a valid option, but not sure we do that for say "subs".
Also, you could have:
```
acc, err := decodeString(w, r, "acc") // Or acc := r.URL.Query().Get("acc") if we don't want to check for "errors"
if err != nil {
    return
}
var opts *LeafzOptions
if subs || acc != "" {
    opts = &LeafzOptions{subs, acc}
}
...
```
I know, saving creating `LeafzOptions` is likely not going to make any difference, but..

Reply:
Space wise this could even be better. (unless the compiler does tricks)
It's on the heap anyway. If you store the result of the lookup in acc, and later on in LeafzOptions, the pointer to the string could be on the heap twice. 
This avoids extra ifs at the cost of computing a pointer value. 

As for r.URL.Query().Get("acc"), we have decode functions in cases where we need to convert the string into some other type. The error then is about conversion issues. 
Here, we do not do a conversion. The returned value is the empty string, which means do not filter, or not, meaning only provide leaf nodes by this name.

Reply:
I am ok with keeping this the way it is.